lib/matplotlib/tests/test_basic.py

```diff
@@ -34,3 +37,36 @@ def test_override_builtins():
 
 def test_verbose():
     assert isinstance(matplotlib.verbose, matplotlib.Verbose)
+
+
+def test_lazy_imports():
+    # The list of modules we want to ensure aren't imported by default
+    lazy_modules = [
+        'matplotlib._png',
+        'matplotlib._tri',
+        'matplotlib._qhull',
+        'matplotlib._contour'
+    ]
+
+    # The list of modules to import that shouldn't import any of the
+    # lazy_modules
+    top_level_modules = [
+        'matplotlib.figure',
+        'matplotlib.backend_bases',
+        'matplotlib.pyplot'
+    ]
+
+    for top_level_module in top_level_modules:
```

Comment:
Just importing all three of them in a single go seems just as good?  This way you don't even need to bother with any formatting...